conans/test/functional/toolchains/cmake/cmakedeps/test_cmakedeps.py

```diff
@@ -661,6 +661,37 @@ def package_info(self):
     assert "talk: Release!" in client.out
     assert "main: Debug!" in client.out
 
+@pytest.mark.tool("cmake", "3.21")
```

Comment:
I think `3.21` is not installed in CI

```suggestion
@pytest.mark.tool("cmake", "3.23")
```